tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationView.java

```diff
@@ -327,6 +385,48 @@ else if( element instanceof MPNode )
         return null;
     }
 
+    private void openBrowser( final String url )
+    {
+        Display.getDefault().asyncExec( new Runnable()
+        {
+
+            public void run()
+            {
+                try
+                {
+                    Desktop desktop = Desktop.isDesktopSupported() ? Desktop.getDesktop() : null;
```

Comment:
Lets not use the Desktop API instead just use the WebBrowserSupport API from Eclipse